### ISSUE TITLE
`bcompare` not in scope?

### ISSUE THREAD

**Issue by Kappie** [bos/criterion#66]
When I run

``` haskell
import Criterion.Main

main = defaultMain [
  bcompare [
    bench "exp" $ whnf exp (2 :: Double)
    , bench "log" $ whnf log (2 :: Double)
    , bench "sqrt" $ whnf sqrt (2 :: Double)
  ]
]
```

I get:

``` bash
 Not in scope: `bcompare'
```

**Comment by ndmitchell** [comment]
I just got confused by this as you are still shipping it as an example. I fixed it by dropping the `bcompare` in #72.

**Comment by AaronFriel** [comment]
Will `bcompare` return?
